server/plugin.go

```diff
@@ -828,6 +828,7 @@ func (p *Plugin) refreshToken(userInfo *gitlab.UserInfo, token *oauth2.Token) (*
 
 	if err != nil {
 		if strings.Contains(err.Error(), "\"error\":\"invalid_grant\"") {
+			p.client.Log.Error("Token revoked while refreshing the token", "userInfo", userInfo, "error", err.Error())
 			p.handleRevokedToken(userInfo)
```

Comment:
```suggestion
			p.client.Log.Warn("Failed to refreshing OAuth token as the existing one is has an invalid grant. Revoking the token.", "userInfo", userInfo, "error", err.Error())
			p.handleRevokedToken(userInfo)
```

Reply:
```suggestion
			p.client.Log.Error("Failed to refresh OAuth token as the existing one has an invalid grant. Revoking the token.", "userInfo", userInfo, "error", err.Error())
			p.handleRevokedToken(userInfo)
```

Reply:
@hanzei Is this confirmed that this will always be a case of invalid grant?

Reply:
We are logging the error now. I hope that will help us figure out :wink: